src/Maestro/Client/src/MaestroApi.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.DotNet.Maestro.Client
+{
+    partial class MaestroApiResponseClassifier
+    {
+        public override bool IsRetriableException(Exception exception)
```

Comment:
This isn't a useful class...

Reply:
I kind of deleted the other conditions by mistake. Readding the missed conditions.